lhotse/cut.py

```diff
@@ -186,11 +209,13 @@ def pad(self, desired_duration: Seconds) -> AnyCut:
 
     @staticmethod
     def from_dict(data: dict) -> 'Cut':
-        feature_info = data.pop('features')
-        supervision_infos = data.pop('supervisions')
+        features = Features.from_dict(data.pop('features')) if 'features' in data else None
+        recording = Recording.from_dict(data.pop('recording')) if 'recording' in data else None
```

Comment:
I think this doesn't work when the dict has `{..., 'features': None, ...}`? Can you add a unit test for that case (same with `recording`?

Reply:
I made two unit tests (`test_load_none_feats_cut_set`, `test_load_none_recording_cut_set`) and those are passed ...